qemu/tests/balloon_check.py

```diff
@@ -25,8 +25,8 @@ def __init__(self, test, params, env):
 
         self.vm = env.get_vm(params["main_vm"])
         self.session = self.get_session(self.vm)
-        logging.info("Waiting 90s for guest's applications up")
-        time.sleep(90)
+        logging.info("Waiting 180s for guest's applications up")
+        time.sleep(180)
```

Comment:
Linux guest won't spend so much time to get into "stable" status, so we need to distinguish the two types of guest if choose this approach.